src/test/java/delete_event_use_case/DeleteEventControllerTest.java

```diff
@@ -0,0 +1,26 @@
+package delete_event_use_case;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+/**
+ * Unit Testing for DeleteEventController class
+ * @author : Khizer Ahmad
+ * @created: November 19th 2022
+ * @since : 2022-11-21, Mon.
+ **/
```

Comment:
Please move the documentation above the package to maintain consistency throughout everyone's files.